com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/PyFunctionMultiTransform/PyFunctionMultiTransform_cpp.cgt

```diff
@@ -130,13 +130,30 @@ void MY_OPERATOR::process(Tuple const & tuple, uint32_t port)
   // convert spl blob to bytes
   PyObject * pyBytes  = streamsx::topology::Splpy::pyBlobToBytes(value);
 
-  // invoke python application function and get an iterator of pickled values
-  PyObject * funcArg = PyTuple_New(2);
-  // first argument is application function
-  Py_INCREF(function_);
-  PyTuple_SetItem(funcArg, 0, function_);
-  // second argument is spl blob in bytes
+  // prepare to call the python utility function that invokes
+  // the application function and returns an iterator
+  PyObject * funcArg = PyTuple_New(3);
+  
+  // first argument is application function or serialized callable class
+  <% if ($serializedClass) { %>
+    PyTuple_SetItem(funcArg, 0, Py_BuildValue("s", <%=$serializedClass%>));

```

Comment:
1) Fixed so that deserializing the instance of the callable class is done once in the constructor.
2) Changed comments to state that we are serializing an instance of a class.